tests/suppress_size.rs

```diff
@@ -0,0 +1,24 @@
+use indoc::indoc;
+
+mod utils;
+
+#[test]
+fn suppress_size() {
+    assert_eq!(
+        utils::run_cmd(&["--suppress-size", "tests/data"]),
```

Comment:
So the test is failing because the output isn't deterministic unless you provide the `--sort` option. I recommend adding `--sort name` to get deterministic output so that tests pass :]